docs/tools/tools_intro.md

```diff
@@ -36,9 +36,9 @@ We created the Mbed command-line tool (Mbed CLI), a Python-based tool, specifica
 
 Mbed OS 5 can be built with various toolchains. The currently supported versions are:
 
-- Arm Compiler 6.11 (default ARM toolchain).
-  - A paid version is available as [Arm Compiler 6.11 Professional](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-6).
-  - A paid version is also included in [Keil MDK 5.28a](http://www2.keil.com/mdk5/).
+- Arm Compiler 6.13 (default ARM toolchain).
+  - A paid version is available as [Arm Compiler 6.13 Professional](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-6).
+  - A paid version is also included in [Keil MDK 5.29](http://www2.keil.com/mdk5/529).
 - [Arm Compiler 5.06 update 6 (to be deprecated in the future)](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler-5/downloads).
 - [GNU Arm Embedded version  6 (6-2017-q1-update)](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads).
```

Comment:
This should be updated then accordingly, however - none of the teams have ARM GCC ver 9 items in the backlog (I would recommend just renaming the existing ones, rather than reject those and create new ones).